cmd/balance.go

```diff
@@ -28,18 +28,18 @@ func balanceCmd() *cobra.Command {
 	cmd.Flags().StringVar(&rpcAddress, "rpc-address", "", "Ethereum RPC Address")
 	cmd.Flags().StringVar(&ics20BankAddress, "ics20-bank-address", "", "Ics20Bank contract address")
 	cmd.Flags().StringVar(&walletAddress, "wallet-address", "", "Wallet address")
-	cmd.Flags().StringVar(&tokenAddress, "token-address", "", "Token address")
+	cmd.Flags().StringVar(&denom, "denom", "", "Token denom")
 
 	cmd.MarkFlagRequired("rpc-address")
 	cmd.MarkFlagRequired("ics20-bank-address")
 	cmd.MarkFlagRequired("wallet-address")
-	cmd.MarkFlagRequired("token-address")
+	cmd.MarkFlagRequired("denom")
 
 	return cmd
 }
 
-func balanceOf(rpcAddress, ics20BankAddress, walletAddress, tokenAddress string) (*big.Int, error) {
-	baseDenom := strings.ToLower(tokenAddress)
+func balanceOf(rpcAddress, ics20BankAddress, walletAddress, denom string) (*big.Int, error) {
+	baseDenom := strings.ToLower(denom)
```

Comment:
Basically OK. BTW why do you use the name `baseDenom` instead of just `denom`?

Reply:
Is `strings.ToLower` needed?

Reply:
It's not necessarily correct to apply `strings.ToLower` for port-id and channel-id.

Reply:
Fix variable name to `denom`
https://github.com/datachainlab/ethereum-ics20-cli/pull/4/commits/882f5f0dd56ebc20b7c908f7b1b2e6ebb086e51b